Lab-4.adoc

```diff
@@ -1,7 +1,388 @@
-= Container Adoption Lab - Module 4
+= Container Adoption Lab
+
+== Module 4
 
 === Expected Outcome:
 
+. Install the petstore application to your Kubernetes cluster
+. Show how the application is self healing
+
 === Lab Requirements:
 
-=== Average Lab Time:
\ No newline at end of file
+. Kubernetes cluster
+. AWS RDS Database
+
+=== Average Lab Time: 20-30 minutes
+
+=== The Lab
+
+==== Overview
+
+Just like you did in link:../Lab-2.adoc[Lab 2] where you created a
+docker-compose.yaml file to deploy your containerized application to Docker for
+X in this module we will create the necessary configurations to deploy the
+petstore into a Kubernetes cluster running in AWS connecting to an RDS database.
+
+Before we get started making the deploy scripts for running the petstore on
+Kubernetes, we're going to talk about Kubernetes basics.
+
+==== Kubernetes Basics
+
+// TODO: Do we need to do any introduction into k8s?
```

Comment:
I think we should do like a minimal intro to what k8s is and what it does. very 101

Reply:
Awesome, will make another issue and fill this out.